convolution/convolution_F_2_64/task.md

```diff
@@ -0,0 +1,47 @@
+## @{keyword.statement}
+
+@{lang.en}
+Given sequences $a_0, a_1, ..., a_{N - 1}$ and $b_0, b_1, ..., b_{M - 1}$ of elements of $\mathbb{F}_{2^{64}}$. Calculate a sequence $c_0, c_1, ..., c_{(N - 1) + (M - 1)}$ of elements of $\mathbb{F}_{2^{64}}$ as follows:
+
+$$c_k = \sum_{i+j=k} a_ib_j$$
+
+
+Let $P(x) = x^{64} + x^4 + x^3 + x + 1 \in \mathbb{Z}_2[x]$ be an irreducible polynomial over $\mathbb{Z}_2$ of degree $64$.
+$\mathbb{F}_{2^{64}}$ is constructed as quotient ring $\mathbb{Z}_2[x] / (P(x))$.
+Elements of $\mathbb{F}_{2^{64}}$ can be uniquely represented by their remainders modulo $P(x)$.
```

Comment:
```suggestion
Let $P(x) = x^{64} + x^4 + x^3 + x + 1 \in \mathbb{Z}_2[x]$ be an irreducible polynomial over $\mathbb{Z}_2$ of degree $64$.
$\mathbb{F} _ {2^{64}}$ is constructed as quotient ring $\mathbb{Z}_2[x] / (P(x))$.
Elements of $\mathbb{F} _ {2^{64}}$ can be uniquely represented by their remainders modulo $P(x)$.
```

Statement will be broken by these `_`'s. This can be fixed by inserting whitespaces.